### PR TITLE
Update data.json 

### PR DESCRIPTION
**Pull request by arwazkhan189**

### CODE REVIEW

File: data.json [hunk 1]
```diff
@@ -298,8 +298,12 @@
...
-    }
-
-
+    }, 
+    {
+        "name": "Arwaz Khan",
+        "gitUsername": "arwazkhan189",
+        "imageUrl": "https://user-images.githubusercontent.com/48387278/96334933-121b9800-1092-11eb-8904-a88502c8b003.png",
+        "issuenumber": 147
+    },
```
Comment (shajeen) [inline data.json:307]: adding comma at the end will break the code.

**Comment by arwazkhan189** [comment]
@shajeen  now please check the same.

**Review by shajeen — APPROVED** [review]
looks good.

**Review by AllanRegush — COMMENTED** [review]
Could you move your profile entry to the end of the json file?

**Comment by AllanRegush** [comment]
Good Work :+1: Just a small adjustment

**Comment by arwazkhan189** [comment]
@AllanRegush  done 👍

File: data.json [hunk 2]
```diff
@@ -298,11 +298,17 @@
         "gitUsername": "frankiefab100",
         "imageUrl": "https://user-images.githubusercontent.com/46662771/96289652-5731c280-0fdd-11eb-8b7f-1eed9d020903.JPG",
         "issuenumber": 133
-    },
+    },
```
Comment (AllanRegush) [inline data.json:301]: Could you remove the trailing white space here?

**Review by AllanRegush — COMMENTED** [review]
Looking Good :+1: Just a small adjustment :smile:

**Comment by arwazkhan189** [comment]
@AllanRegush  done 👍

**Comment by AllanRegush** [comment]
Awesome :+1: Looks good! Thank you.

**Pull request merged**